openpype/lib/applications.py

```diff
@@ -1917,6 +1917,13 @@ def _prepare_last_workfile(data, workdir, modules_manager):
     data["env"]["AVALON_LAST_WORKFILE"] = last_workfile_path
     data["last_workfile_path"] = last_workfile_path
 
+    # If last workfile path is found, don't launch workfile tool
+    if start_last_workfile and last_workfile_path:
+        data["env"]["OPENPYPE_WORKFILE_TOOL_ON_START"] = "0"
+        log.debug(
+            "Last workfile path found so workfile tool won't be launched."
+        )
```

Comment:
I'd actually defer this to the launching of the tool on startup itself - to then check if `AVALON_LAST_WORKFILE` was set?

Reply:
same as my other comment, I think it makes more sense if the logic is in the common code of all hosts than implementing it per host?